### ISSUE TITLE
update year in license.

### ISSUE THREAD

**Issue by bharatpurohit97** [coala/documentation#582]
In the footer presently it is `©2017, The coala Developers`

**Comment by anand372** [comment]
Can I be issued this issue

**Comment by jayantsudo24** [comment]
Hello I am interested to pick up this issue <https://github.com/coala/documentation/issues/582> ,

**Comment by jayantsudo24** [comment]
Okay, actually corobo was down I don't know for some reason.. thanks for informing

**Comment by CalciumCat** [comment]
Hi :) Does thi still need fixing?

**Comment by bharatpurohit97** [comment]
@CalciumCat to work on any issue  you need to be `assigned` to the issue as mentioned in [Newcomers Guide](https://api.coala.io/en/latest/Developers/Newcomers_Guide.html), and First you need to be a `member`, ping maintainers for invitation. Do not make PR before.

**Comment by bharatpurohit97** [comment]
@Jain-Aditya ask on channel to get `assigned`, and first you need to be member ask for invitation to maintainers as well.

**Comment by sambhavkaul** [comment]
Hey, is this issue still open ? Can I work on it ?

**Comment by budukhyash** [comment]
@sambhavkaul Yes this issue is open.Please contact here https://gitter.im/coala/coala to get it assigned.
Ping me if you need any help.

**Comment by sambhavkaul** [comment]
@budukhyash I already did but no one responded. Thanks btw

**Comment by budukhyash** [comment]
@sambhavkaul wait for the maintainers to come online.They will surely assign it to you

**Comment by dev-313** [comment]
Can you please assign me this issue ?
